src/grimoirelab/core/scheduler/scheduler.py

```diff
@@ -106,6 +106,48 @@ def cancel_task(task_uuid: str) -> None:
     task.delete()
 
 
+def update_task(
```

Comment:
I don't think we need this now. Either the repository is available or not. Let's keep it as simple as that. We will change the API a lot so, we can wait until we add this kind of operations.